gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -0,0 +1,60 @@
+package com.gravatar.quickeditor.ui.editor
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import com.gravatar.quickeditor.ui.oauth.OAuthPage
+import com.gravatar.quickeditor.ui.oauth.OAuthParams
+
+@Composable
+internal fun GravatarQuickEditorPage(
```

Comment:
Internal for now, though we can open it later. That will open the possibility of using the quick editor flow in a new activity for example.